nixos/roles/gitlab.nix

```diff
@@ -44,28 +44,55 @@ in
       (writeScriptBin "gitlab-show-config" ''jq < /srv/gitlab/state/config/gitlab.yml'')
     ];
 
+    # all logs to /var/log
+    systemd.tmpfiles.rules = [
```

Comment:
cfg is wrong here, cfg is defined as config.flyingcircus.roles.gitlab in the role

you want config.services.gitlab here